src/NuGet.Core/NuGet.Build.Tasks/Common/MSBuildLogger.cs

```diff
@@ -171,6 +171,9 @@ private void LogForMono(ILogMessage message)
             LogErrorWithDetails logWithDetails,
             LogErrorAsString logAsString)
         {
+            // Work around for https://github.com/Microsoft/msbuild/issues/2316
+            var logMessageString = logMessage.Message.Replace("\r\n", "\n");
```

Comment:
There is already a logMessage and the string indicates that this var only contains the string equivalent of the type `LogMessage`.